data/jobs.txt

```diff
@@ -1483,6 +1483,169 @@ mission "Core Pirate Attack"
+			choice
+				`	"Sure, I always welcome targeting practice."`
+					accept
+				`	"No, sorry, I'm busy right now."`
+					decline
+	npc evade
+		government "Renegade"
+		personality heroic target vindictive nemesis
+			confusion 30
+		fleet
+			names "republic fighter"
+			variant
+				"Combat Drone" 10
+	npc evade
+		government "Renegade"
+		personality waiting heroic vindicitive target
+			confusion 30
+		fleet
+			names "republic fighter"
+			variant
+				"Combat Drone" 30
+	npc
+		personality heroic staying uninterested
+		government "Republic"
+		fleet
+			names "republic fighter"
+			variant
+				"Combat Drone" 20
+	on complete
+		payment 200000
+		dialog "You got the drones under control and collect your payment of <payment>."
+
+mission "Renegade Drones 3"
+	name "Neutralize AI Drones"
+	repeat
+	description "Destroy or disable the malfunctioning Navy drones."
+	minor
+	to offer
+		not "Renegade Drones 1: active"
+		not "Renegade Drones 2: active"
+		not "Renegade Drones 3: active"
+		random < 5
+		"Renegade Drones 2: done" > 3
+		"combat rating" > 1200
+	clearance
+	source "Pilot"
+	destination "Pilot"
+	on offer
+		conversation
+			`You abruptly turn around when a rough but very distinct voice calls your name: "Captain <last>! Been looking for you." You stand in front a senior Navi Captain. "One says, you have quite some experience with drones going berzerk. Apparently, those guys from the lab meant it a little bit to well with the artificial intelligence in our newest drone model. It seems it started wondering, why it should work for us. Would you be willing to, let's say, correct their mistake?"`
+			choice
+				`	"Preventing the rise of the machines? Would be hard to say no."`
+					goto yes
+				`	"I'm sorry, I try not to kill sentient beings."`
+					decline
+			label yes
+			`"Great! A group of Combat Drones will support you."`
+				accept
+	npc evade
+		government "Renegade"
+		personality heroic target nemesis
+			confusion 0
+		fleet
+			names "female names"
+			variant
+				"Heavy Combat Drone" 20
+	npc evade
+		government "Renegade"
+		personality waiting heroic vindictive target
+			confusion 0
+		fleet
+			names "male names"
+			variant
+				"Heavy Combat Drone" 50
+	npc
+		government "Republic"
+		personality waiting heroic uninterested
+		fleet
+			names "republic fighter"
+			variant
+				"Combat Drone" 70
+	on complete
+		payment 400000
+		dialog "Collecting your payment of <payment>, you still wonder if it was the right choice to destroy an emerging intelligence."
+
+government "Renegade"
+	swizzle 0
+	"player reputation" -1000
+	"attitude toward"
+		# Broken drones don't like anybody
+		"Merchant" -.1
+		"Pirate" -.1
+		"Bounty Hunter" -.1
+		"Republic" -.5
+		"Navy Intelligence" -.5
+		"Navy (Oathkeeper)" -.5
+	"penalty for"
```

Comment:
Redundant information; drones have no crew and therefore yield no penalty for attacking them.